docs/cudf/source/conf.py

```diff
@@ -50,7 +50,7 @@
     "myst_nb",
 ]
 
-jupyter_execute_notebooks = "force"
+jupyter_execute_notebooks = "cache"
```

Comment:
@mmccarty - any reason we chose not to cache notebook execution here?

Reply:
If we use `cache` or `auto` here, the notebooks will only be executed with they are updated. `force` will ensure they are executed more frequently when we build the docs so that we make sure they are up-to-date with `cudf` itself.

Reply:
That makes sense -- thanks! I'll undo this change then.